manim/animation/animation.py

```diff
@@ -238,7 +238,7 @@ def __init__(
         self.duration = duration
         self.mobject = None
         self.stop_condition = stop_condition
-        super().__init__(None, **kwargs)
+        super().__init__(Mobject(), **kwargs)
```

Comment:
Question : is this changed intended? it seems like off-topic

Reply:
Intended, I updated the repo to recent community version changes